.travis.yml

```diff
@@ -16,6 +16,7 @@ before_install:
 # Install packages
 install:
   - conda install --yes python=$TRAVIS_PYTHON_VERSION numba matplotlib scipy h5py
+  - conda install --yes gcc 
```

Comment:
This is unrelated to the rest of the pull request, but was necessary to get the tests to pass.
(see https://github.com/conda-forge/mpi4py-feedstock/issues/1)